nms/packages/magmalte/app/views/equipment/EnodebDetailSummaryStatus.js

```diff
@@ -122,6 +122,59 @@ export function EnodebStatus({refresh}: {refresh: boolean}) {
         value: enbInfo?.enb_state.ip_address ?? 'Not Available',
       },
     ],
+    [
+      {
+        category: 'SOFTWARE Version',
```

Comment:
Why is `SOFTWARE` all upper case?

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812817-fc90106b-6443-44f3-a379-38392be234f6.png)
modified,thank you!